src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -28,7 +28,9 @@ export default () => (
       className="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row"
     >
       <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
-        <b>File a VA disability claim</b>
+        <h3 className="vads-u-font-size--base vads-u-margin--0">
```

Comment:
How come we're making these headers if they don't look like headers out of curiosity?

Reply:
Navigation and semantic purposes from the a11y team I believe.